### PR TITLE
Profile details

### PR DESCRIPTION
**Pull request by TalReinfeld**
Adding a html file (profile_details) that extends profile.html,
the file show all the profile details.

Adding css file for the above.

Adding a test to check the file content.

Changing the urls for the new pages. - http://127.0.0.1:8000/profile/settings/

This PR is depends on PR #127

Solve issue #107 

image

image

image

### CODE REVIEW

**Review by idoyek — APPROVED** [review]
LGTM

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by idosinger98 — APPROVED** [review]

File: account/views/client_views.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
 def show_profile(request, client_id):
     client = get_object_or_404(Client, pk=client_id)
     profile = client.profile_id
-    return render(request, 'account/profile.html', {'client': client, 'profile': profile})
+    return render(request, 'account/profile_details.html', {'client': client, 'profile': profile})
```
Comment (Yarboa) [inline account/views/client_views.py:8]: This parameter could be retrieved through
 response.context.get("client")
 response.context.get("profile")

**Review by Yarboa — CHANGES_REQUESTED** [review]
Please review comments
You do not assert the content of the rersponses

**Review by idosinger98 — CHANGES_REQUESTED** [review]
Please use base.html

**Comment by TalReinfeld** [comment]
> Please use base.html

@idosinger98 Done

**Review by idosinger98 — COMMENTED** [review]
please convert to draft

**Review by idosinger98 — COMMENTED** [review]
please convert to draft

**Comment by Yarboa** [comment]
I think you need to rebase

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**